testnets/outbedevnet/assetlist.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://github.com/cosmos/chain-registry/blob/master/assetlist.schema.json",
```

Comment:
The schema locations are required to be relative locations (so like ../../chain.schema.json)